### PR TITLE
Post-CMA improvements

### PR DESCRIPTION
**Pull request by filipzeta**
A couple of extra bits:
- improve MA -> CMA migration
- extra backend bits in IDL

### CODE REVIEW

File: src/cross-client.ts [hunk 1]
```diff
@@ -616,12 +616,39 @@ export class CrossClient {
...
+          Exchange.programId,
+          Exchange.getZetaGroupAddress(asset),
+          this.publicKey
+        );
+
+        // Check if the address is valid
+        let account =
+          Exchange.program.account.marginAccount.fetchNullable(address);
+
+        if (account != null) {
```
Comment (kelvinlau20100) [inline src/cross-client.ts:635]: nit; can't this just be `if (account)`

File: src/risk-utils.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { BN } from "@zetamarkets/anchor";
-import { types, Exchange, constants, assets, instructions } from ".";
+import { types, Exchange, constants, assets, instructions, utils } from ".";
```
Comment (kelvinlau20100) [inline src/risk-utils.ts:2]: is this import needed no other code was added?

Comment (filipzeta) [reply src/risk-utils.ts]: removed

File: src/cross-client.ts (hunk 1 above)
Comment (filipzeta) [reply src/cross-client.ts]: fixed

**Review by kelvinlau20100 — APPROVED** [review]

**Pull request merged**